admin/views/tabs/social/accounts.php

```diff
@@ -69,9 +69,18 @@
 	echo '<p>';
 	$user_id = WPSEO_Options::get( 'company_or_person_user_id', '' );
 	$person  = get_userdata( $user_id );
-	printf( esc_html__( 'To change the social accounts used for your site, update the details for %1$s.', 'wordpress-seo' ), '<a href="' . admin_url( 'user-edit.php?user_id=' . $user_id ) . '">' . $person->display_name . '</a>' );
+	printf(
+		/* translators: 1: link to edit user page. */
```

Comment:
The wording of the translator comment is different from the wording in the `knowledge-graph.php` file; there `%1$s` is used, here only `1`. Also, this comment is formulated as an imperative, the other one as a 3rd person verb.

Reply:
@johannadevos Ok, so there is no sniff yet to check on the content of translator comments (darn, I'll to write that at some point), however, there are guidelines on how translator comments should be formatted.

When the text string contains only one placeholder, it will normally not be numbered, like `%s` or `%d`. In that case, the translators comment can just reference the placeholder, like so:
```php
/* translators: %s: something */
```

When the text string contains multiple placeholders, they should be numbered (there is already a sniff which checks that).
In the translator comment the numbers should be used in that case, not the placeholder (even when there is only one placeholder, but it's numbered) . The numbers should be listed in order and separated by a semi-colon, like so:
```php
/* translators: 1: something; 2: something else */
```

So the comment in the `knowledge-graph.php` file should be improved, but the comment I added here is fine (format wise).

Regarding the language - imperative vs 3rd person -, I'm not sure if there is a strict guideline, however, imperative feels more appropriate.

The comment is supposed to tell the translator what the placeholder will be replaced with in the text, so the translator knows how to handle the placeholder for the sentence-order.

The translators comment in `knowledge-graph.php` in that sense feels wrong as it talks about the replacements as if they are actions (which they are not at the time of translation).
```php
/* translators: %1$s opens the link to the Yoast.com article about Google's Knowledge Graph, %2$s closes the link, */
```

Improving existing translators comments to follow these guidelines would be a great idea, but is outside of the scope of this PR.

I propose we leave the one in `knowledge-graph.php` as-is for the moment and do a review of all translators comments at some point. Maybe a `technical-debt` ticket should be opened for this ?

If nothing else, once a sniff has been written to check this, this will be reported on by PHPCS.